frontend/src/components/Table.jsx

```
@@ -0,0 +1,31 @@
+import { Edit, Eye, Trash2 } from "lucide-react";
+
+const EmployeeTable = ({ renderData, colData }) => {
+  return (
+    <div className="bg-white rounded-2xl shadow-sm border-gray-200 overflow-hidden mt-5">
+      <div className="overflow-x-auto">
+        <table className="w-full border-collapse">
+          <thead className="bg-gray-50 border-b border-gray-200 ">
+            <tr>
+              {colData.map((column) => (
+                <td className={column.className}>
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)